dashboard/app/models/script_level.rb

```diff
@@ -158,10 +158,7 @@ def final_level?
   end
 
   def next_level_or_redirect_path_for_user(user, extras_lesson=nil)
-    if bubble_choice?
-      # Redirect user back to the BubbleChoice activity page.
-      level_to_follow = self
```

Comment:
Hey, I wanted to double check - is the "return to parent level" functionality still available with this change? i.e. it looks like this line was originally added here: https://github.com/code-dot-org/code-dot-org/pull/29075

Reply:
good catch. there is also a test for this that is currently failing. i will sync with Maddie on how to resolve.